<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>WindowMain</class>
	<widget class="QMainWindow" name="WindowMain">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>230</width>
				<height>309</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Developer</string>
		</property>
		<property name="windowIcon">
			<iconset>
				<normaloff>../icons/ToolSeq.svg</normaloff>../icons/ToolSeq.svg
			</iconset>
		</property>
		<widget class="QWidget" name="Widget_00">
			<layout class="QVBoxLayout" name="verticalLayout">
				<property name="spacing">
					<number>2</number>
				</property>
				<property name="leftMargin">
					<number>2</number>
				</property>
				<property name="topMargin">
					<number>2</number>
				</property>
				<property name="rightMargin">
					<number>2</number>
				</property>
				<property name="bottomMargin">
					<number>2</number>
				</property>
				<item>
					<widget class="QWidget" name="Widget_01" native="true">
						<layout class="QVBoxLayout" name="verticalLayout_2">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_011" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_5">
										<property name="spacing">
											<number>2</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QRadioButton" name="Radio_Attributes">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Attributes</string>
												</property>
												<attribute name="buttonGroup">
													<string notr="true">ButtonGroup_Mode</string>
												</attribute>
											</widget>
										</item>
										<item>
											<widget class="QCheckBox" name="Check_Writable">
												<property name="enabled">
													<bool>false</bool>
												</property>
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Writable</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="QLineEdit" name="Line_Attributes">
									<property name="enabled">
										<bool>false</bool>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QRadioButton" name="Radio_CommandsMel">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Commands + Procedures [ mel ]</string>
									</property>
									<property name="checked">
										<bool>true</bool>
									</property>
									<attribute name="buttonGroup">
										<string notr="true">ButtonGroup_Mode</string>
									</attribute>
								</widget>
							</item>
							<item>
								<widget class="QRadioButton" name="Radio_CommandsPython">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Commands [ python &gt; maya.cmds ]</string>
									</property>
									<attribute name="buttonGroup">
										<string notr="true">ButtonGroup_Mode</string>
									</attribute>
								</widget>
							</item>
							<item>
								<widget class="QRadioButton" name="Radio_VariablesMel">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Global Variables [ mel ]</string>
									</property>
									<attribute name="buttonGroup">
										<string notr="true">ButtonGroup_Mode</string>
									</attribute>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="Line" name="Widget_02">
						<property name="sizePolicy">
							<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
								<horstretch>0</horstretch>
								<verstretch>0</verstretch>
							</sizepolicy>
						</property>
						<property name="minimumSize">
							<size>
								<width>0</width>
								<height>10</height>
							</size>
						</property>
						<property name="frameShadow">
							<enum>QFrame::Plain</enum>
						</property>
						<property name="lineWidth">
							<number>2</number>
						</property>
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_03" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_2">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLineEdit" name="Line_Search">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_04" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_6">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QCheckBox" name="Check_CaseSensitive">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Case Sensitive</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QCheckBox" name="Check_Regex">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Fixed">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Regex</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_05" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPushButton" name="Button_Search">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Search</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="Button_Act">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="text">
										<string>Help</string>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_06" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_3">
							<property name="spacing">
								<number>2</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QPlainTextEdit" name="Text_Results">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Ignored" vsizetype="Expanding">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>25</height>
										</size>
									</property>
									<property name="lineWrapMode">
										<enum>QPlainTextEdit::NoWrap</enum>
									</property>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<widget class="QWidget" name="Widget_07" native="true">
						<layout class="QHBoxLayout" name="horizontalLayout_8">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>0</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<property name="bottomMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QWidget" name="Widget_071" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_4">
										<property name="spacing">
											<number>10</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="label_7">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>All</string>
												</property>
												<property name="alignment">
													<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_2">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="Label_CountAll">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>0</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
							<item>
								<widget class="Line" name="line">
									<property name="sizePolicy">
										<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
											<horstretch>0</horstretch>
											<verstretch>0</verstretch>
										</sizepolicy>
									</property>
									<property name="minimumSize">
										<size>
											<width>10</width>
											<height>25</height>
										</size>
									</property>
									<property name="frameShadow">
										<enum>QFrame::Plain</enum>
									</property>
									<property name="lineWidth">
										<number>2</number>
									</property>
									<property name="orientation">
										<enum>Qt::Vertical</enum>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QWidget" name="Widget_072" native="true">
									<layout class="QHBoxLayout" name="horizontalLayout_7">
										<property name="spacing">
											<number>10</number>
										</property>
										<property name="leftMargin">
											<number>0</number>
										</property>
										<property name="topMargin">
											<number>0</number>
										</property>
										<property name="rightMargin">
											<number>0</number>
										</property>
										<property name="bottomMargin">
											<number>0</number>
										</property>
										<item>
											<widget class="QLabel" name="Label_CountFiltered">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>0</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label_6">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Fixed" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>:</string>
												</property>
											</widget>
										</item>
										<item>
											<widget class="QLabel" name="label">
												<property name="sizePolicy">
													<sizepolicy hsizetype="Ignored" vsizetype="Ignored">
														<horstretch>0</horstretch>
														<verstretch>0</verstretch>
													</sizepolicy>
												</property>
												<property name="minimumSize">
													<size>
														<width>0</width>
														<height>25</height>
													</size>
												</property>
												<property name="text">
													<string>Filtered</string>
												</property>
											</widget>
										</item>
									</layout>
								</widget>
							</item>
						</layout>
					</widget>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections>
		<connection>
			<sender>Radio_Attributes</sender>
			<signal>toggled(bool)</signal>
			<receiver>Line_Attributes</receiver>
			<slot>setEnabled(bool)</slot>
			<hints>
				<hint type="sourcelabel">
					<x>39</x>
					<y>52</y>
				</hint>
				<hint type="destinationlabel">
					<x>114</x>
					<y>80</y>
				</hint>
			</hints>
		</connection>
		<connection>
			<sender>Radio_Attributes</sender>
			<signal>toggled(bool)</signal>
			<receiver>Check_Writable</receiver>
			<slot>setEnabled(bool)</slot>
			<hints>
				<hint type="sourcelabel">
					<x>39</x>
					<y>52</y>
				</hint>
				<hint type="destinationlabel">
					<x>154</x>
					<y>53</y>
				</hint>
			</hints>
		</connection>
	</connections>
	<buttongroups>
		<buttongroup name="ButtonGroup_Mode"/>
	</buttongroups>
</ui>